Regression/WarpX-tests.ini

```diff
@@ -1034,3 +1034,36 @@ compileTest = 0
 doVis = 0
 compareParticles = 0
 analysisRoutine = Examples/Tests/collision/analysis_collision.py
+
+# NOTE: input file previously not tested, now testing only run (no analysis)
```

Comment:
```suggestion
```
We have a number of other tests like this one, so I'm not sure we need this precision for these 2.